crypto/x509v3/v3_lib.c

```diff
@@ -156,6 +156,7 @@ static const X509V3_EXT_METHOD *standard_exts[] = {
     &v3_ct_scts[0],
     &v3_ct_scts[1],
 #endif
+	&v3_tls_feature,
```

Comment:
Sorry, it was a stray TAB.  Fixed in https://github.com/robstradling/openssl/commit/547b5efd03c734bc0317ce4a93a8befcc6d47d56